modules/abstract-cosmos/src/cosmosCoin.ts

```diff
@@ -351,10 +351,10 @@ export class CosmosCoin extends BaseCoin {
     if (!rawTx) {
       throw new Error('missing required tx prebuild property txHex');
     }
-    const transaction = await this.getBuilder().from(rawTx).build();
-    const explainedTx = transaction.explainTransaction();
 
     if (txParams.recipients && txParams.recipients.length > 0) {
+      const transaction = await this.getBuilder().from(rawTx).build();
+      const explainedTx = transaction.explainTransaction();
       const filteredRecipients = txParams.recipients?.map((recipient) => _.pick(recipient, ['address', 'amount']));
```

Comment:
i think leaving this outside of the if condition provides a validation since parsing the tx its also a validation

Reply:
I'll look at this again as a part of https://bitgoinc.atlassian.net/browse/COIN-3429